.editorconfig

```diff
@@ -424,4 +424,11 @@ dotnet_diagnostic.IDE0100.severity = suggestion
 dotnet_diagnostic.IDE0073.severity = error
 
 # Use 'System.Threading.Lock'
-dotnet_diagnostic.IDE0330.severity = suggestion
\ No newline at end of file
+dotnet_diagnostic.IDE0330.severity = suggestion
+
+
+# xunit to supress temp
+dotnet_diagnostic.xUnit2020.severity = none
+dotnet_diagnostic.xUnit1031.severity = none
+dotnet_diagnostic.xUnit1012.severity = none
+dotnet_diagnostic.xUnit2029.severity = none
```

Comment:
We have an existing `xUnit2013` suppression here, can you unify them in one place or the other?

https://github.com/dotnet/msbuild/blob/a03ee4ba9c5e71a361d86a6ef7923c44a0dc5f39/eng/Common.globalconfig#L1107-L1108

Reply:
Moved to .editorconfig thank you for pointing to that